spec/beaker/dsl/structure_spec.rb

```diff
@@ -128,6 +128,14 @@ class ClassMixedWithDSLStructure
       subject.confine( :to, {} )
     end
 
+    it 'uses a provided host subset when no criteria is provided' do
+
+      subset = ['host1', 'host2']
+      hosts = subset << 'host3'
```

Comment:
@anodelman it looks like `hosts` and `subset` are the same after this point, making this test not as helpful:

    ➜  beaker git:(master) ✗ ruby --version
    ruby 1.9.3p484 (2013-11-22 revision 43786) [x86_64-darwin13.0.0]
    ➜  beaker git:(master) ✗ irb
    irb(main):001:0> s1 = [1, 2]
    => [1, 2]
    irb(main):002:0> s2 = s1 << 3
    => [1, 2, 3]
    irb(main):003:0> s2
    => [1, 2, 3]
    irb(main):004:0> s1
    => [1, 2, 3]
    irb(main):005:0> s2.push(4)
    => [1, 2, 3, 4]
    irb(main):006:0> s1
    => [1, 2, 3, 4]

`Array.new` or `compact` look to be good solutions to this issue from my experiments.  Maybe I'm misunderstanding what you're going for here?